components/brave_wallet/renderer/brave_wallet_js_handler.cc

```diff
@@ -107,6 +150,9 @@ v8::Local<v8::Promise> BraveWalletJSHandler::Request(
   if (!out || !out->is_dict() || !out->GetAsDictionary(&out_dict))
     return v8::Local<v8::Promise>();
 
+  // Hardcode id to 1 as it is unused
+  ALLOW_UNUSED_LOCAL(out_dict->SetIntPath("id", 1));
```

Comment:
these should probably be consts defined in brave_wallet/common or something

Reply:
I'm going to do it as consts in that cc as they are not supposed to be used anywhere else